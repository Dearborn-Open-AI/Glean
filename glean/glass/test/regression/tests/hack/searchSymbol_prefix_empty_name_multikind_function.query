action: searchSymbol
args: {"name": "", "repo_name": "test", "language": [3], "kinds": [14], "options": {"detailedResults": false, "exactMatch": false, "ignoreCase": false, "namespaceSearch": false } }
