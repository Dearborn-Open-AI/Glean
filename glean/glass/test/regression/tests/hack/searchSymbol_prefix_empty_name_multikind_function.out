[
    "@generated",
    {
        "symbolDetails": [],
        "symbols": [
            {
                "kind": 14,
                "language": 3,
                "location": {
                    "filepath": "www/TopLevel.php",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 2,
                        "lineBegin": 12,
                        "lineEnd": 14
                    },
                    "repository": "test"
                },
                "name": "corge",
                "symbol": "test/php/corge"
            }
        ]
    }
]